satpy/etc/readers/oci_l2_bgc.yaml

```
@@ -0,0 +1,37 @@
+reader:
+  name: oci_l2_bgc
+  short_name: PACE OCI L2 BGC
+  long_name: PACE OCI L2 Biogeochemical in NetCDF format
+  description: PACE OCI L2 Biogeochemical Reader
+  status: Beta
+  supports_fsspec: false
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [oci]
+
+file_types:
+  bgc_nc:
+    file_patterns:
+    # Example: PACE_OCI.20240907T191809.L2.OC_BGC.V2_0.NRT.nc4
+      - '{platform:s}_{sensor:s}.{start_time:%Y%m%dT%H%M%S}.L2.OC_BGC.V{sw_version:s}.{processing_type:s}nc4'
```

I think a `.` is missing before the file ext `nc4`.
```suggestion
      - '{platform:s}_{sensor:s}.{start_time:%Y%m%dT%H%M%S}.L2.OC_BGC.V{sw_version:s}.{processing_type:s}.nc4'
```

---

Nope. That was intentional in case the last group doesn't exist.

---

thanks for clarifying it.